composeApp/src/commonMain/sqldelight/org/ooni/probe/data/TestDescriptor.sq

```diff
@@ -71,8 +71,7 @@ setAutoUpdate:
 UPDATE TestDescriptor SET auto_update = ? WHERE runId = ?;
 
 selectAll:
-SELECT * FROM TestDescriptor;
-
+SELECT * FROM TestDescriptor WHERE (is_expired IS NULL OR (is_expired = :isExpired AND expiration_date > strftime('%s', 'now'))) OR (:isExpired IS NULL);
```

Comment:
I believe this query is wrong. It should be ignoring the `is_expired` DB column and it doesn't work for returning the expired descriptors. Maybe try writing some tests for it?

I was expecting something like:
```
SELECT * FROM TestDescriptor WHERE
:isExpired IS NULL 
OR (:isExpired = 1 AND expiration_date < strftime('%s', 'now'))
OR (:isExpired = 0 AND expiration_date >= strftime('%s', 'now'))
```